deploy/helm/events-collector-config.yaml

```diff
@@ -166,6 +271,30 @@ service:
         - batch
       receivers:
         - k8s_events
+{{- end }}
+{{- if .Values.otel.events.manifests.enabled }}
+    logs/manifests:
+      exporters:
+        - otlp
+      processors:
+        - memory_limiter
+        - transform/manifest
+        - groupbyattrs/manifest
+        - resource/manifest
+        - k8sattributes
```

Comment:
Does it mean we will annotate the manifests with labels/annotations even though they are already part of the manifests?

Reply:
Yes it annotates with labels and annotations setting them to resource attributes, so that it is mapped to the entity. I don't know how to expand annotations/labels from manifest to resource attributes using OTTL. It is probably possible, but it was easier to leverage k8sattributes for this job.